.github/workflows/cd.yml

```diff
@@ -112,15 +112,10 @@ jobs:
         run: |
           npx lerna publish --no-private --dist-tag=latest ${{ github.event.inputs.version }} --yes ${{ github.event.inputs.force }}
 
-      - name: release rc npm packages to npmjs.org by conventional-commits
-        if: ${{ github.event_name == 'workflow_dispatch' && github.ref == 'refs/heads/main' && github.event.inputs.conventional-commits == 'yes' && github.event.inputs.preid == 'rc'}}
+      - name: publish npm packages to npmjs.org by conventional-commits
+        if: ${{ github.event_name == 'workflow_dispatch' && github.ref == 'refs/heads/main' && github.event.inputs.conventional-commits == 'yes'}}
         run: |
-          npx lerna publish --conventional-prerelease --preid=rc --yes
-
-      - name: release stable npm packages to npmjs.org by conventional-commits
-        if: ${{ github.event_name == 'workflow_dispatch' && github.ref == 'refs/heads/main' && github.event.inputs.conventional-commits == 'yes' && github.event.inputs.preid == 'stable'}}
-        run: |
-          npx lerna publish --conventional-graduate --yes
+          npx lerna publish from-package --yes
```

Comment:
Did we test whether it will release all packages? What is the manual step if we need update the version manually?

Reply:
yes, "from-package" will look inside the latest tag which lerna version put on the branch, and will publish the version tagged on the latest commits.
manually step is:
1. release manager fetch origin main, and run "npm run version-rc"
2. make sure the rc/stable version change, then input "yes" or "y"
3. push to the origin main branch, and trigger the CD pipeline, must choose "conventional-commits" as "yes"